README.md

```diff
@@ -4,7 +4,9 @@
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/ansible-role-remove-python2.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-remove-python2/alerts/)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/ansible-role-remove-python2.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-remove-python2/context:python)
 
-An Ansible role for removing all python2 packages.
+An Ansible role for removing all python2 packages on all distributions
```

Comment:
Nitpick: "python2" should be capitalized.

Reply:
Although now that I look at it again, if it's referring to the package name of "python2" I would prefer if that was more clear.

Reply:
Capitalized in commit 1adfd37.